05-JWT-Basics/Preferred/app.js

```diff
@@ -0,0 +1,26 @@
+require("dotenv").config();
+require("express-async-errors");
+
+const express = require("express");
+const app = express();
+
+app.use(express.static("./public"));
+app.use(express.json());
+
+const mainRouter = require("./routes/main");
```

Comment:
This works and isn't _wrong_ but it's _most_ typical to complete all imports/requires at the top of the file and then have your code (variable declaration, invocations, etc.) come afterwards.